src/container_registry/scaleway_container_registry.rs

```diff
@@ -396,8 +396,8 @@ impl ContainerRegistry for ScalewayCR {
         if !force_push && self.does_image_exists(&image) {
             // check if image does exist - if yes, do not upload it again
             let info_message = format!(
-                "image {:?} found on Scaleway {} repository, container build is not required",
-                image, registry_name,
+                "image {} found on Scaleway {} repository, container build is not required",
```

Comment:
could you add the tag and the URL may be?

Reply:
done